<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceForm</class>
 <widget class="QDialog" name="InvoiceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#dvt, #mvt, #slt, #tvt, #mnv, #ho, #ten, #phai, #dongia_2, #hoten_2, #ngay_2, #loai_2, #tenvattu_2, #soluong_2, #trigia_2, #shd, #nl, #lhd, #htnvl, #tghd, #thang, #doanhthu, #statisticFacilityId, #statisticFacilityName, #statisticFacilityValue,#VATLabel{
                                        background-color: #00A4E8;
                                        color: #ffffff;
                                        font: Bold &quot;Inter&quot;;
                                        }

                                        QPushButton {
                                        font: Bold &quot;Lato&quot;;
                                        color: #fff;
                                        }

                                        QTableWidget{
                                        border: 1px solid gray;
                                        border-radius: 5px;
                                        }

#centralwidget {
                    background-color: #fff
                    }

                    QPushButton {
                    border-radius: 5px;
                    background-color: black;
                    height: 24px;
                    }

                    QTableWidget
                    {
                    background-color: #fff;
                    }

                    #facilityAddButton, #staffAddButton, #invoiceAddButton
                    {
                    background-color: #198754;
                    }

                    #facilityAddButton:hover,#staffAddButton:hover, #invoiceAddButton:hover
                    {
                    background-color: #157347;
                    }

                    #facilityDeleteButton, #staffDeleteButton, #invoiceDeleteButton
                    {
                    background-color: #dc3545;
                    }

                    #facilityDeleteButton:hover,#staffDeleteButton:hover, #invoiceDeleteButton:hover
                    {
                    background-color: #bb2d3b;
                    }

                    #facilityEditButton, #staffEditButton, #invoiceEditButton
                    {
                    background-color: #0d6efd;
                    }

                    #facilityEditButton:hover,#staffEditButton:hover, #invoiceEditButton:hover
                    {
                    background-color: #0b5ed7;
                    }</string>
  </property>
  <widget class="QGroupBox" name="invoiceGroup">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>30</y>
     <width>251</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin hóa đơn</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="invoiceId">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ngày lập</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loại hóa đơn</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="invoiceDate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="invoiceAddButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Thêm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invoiceDeleteButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invoiceEditButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Hiệu chỉnh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="invoiceImport">
       <property name="text">
        <string>Nhập</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="invoiceExport">
       <property name="text">
        <string>Xuất</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu ý: Click đúp vào hoá đơn để mở chi tiết hoá đơn</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>701</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="loai_2">
        <property name="styleSheet">
         <string notr="true">#label {
                                                                           background-color: #00A4E8;
                                                                           color: #ffffff;
                                                                           font: Bold &quot;Inter&quot;;
                                                                           }</string>
        </property>
        <property name="text">
         <string>Loại</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tenvattu_2">
        <property name="styleSheet">
         <string notr="true">#label {
                                                                           background-color: #00A4E8;
                                                                           color: #ffffff;
                                                                           font: Bold &quot;Inter&quot;;
                                                                           }</string>
        </property>
        <property name="text">
         <string>Mã hóa đơn</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ngay_2">
        <property name="styleSheet">
         <string notr="true">#label {
                                                                           background-color: #00A4E8;
                                                                           color: #ffffff;
                                                                           font: Bold &quot;Inter&quot;;
                                                                           }</string>
        </property>
        <property name="text">
         <string>Ngày</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="VATLabel">
        <property name="styleSheet">
         <string notr="true">#label {
                                                                           background-color: #00A4E8;
                                                                           color: #ffffff;
                                                                           font: Bold &quot;Inter&quot;;
                                                                           }</string>
        </property>
        <property name="text">
         <string>Trị giá hoá đơn</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="InvoiceTable">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::PenStyle::DashLine</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
